openassessment/templates/openassessmentblock/staff_area/oa_student_info_assessment_detail.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+{% spaceless %}
```

Comment:
Remove spaceless-- it's causing the lack of space between the collapse/expand icon and the title.

![image](https://cloud.githubusercontent.com/assets/484484/12430238/96ec7856-bebc-11e5-978a-aa694fb35c27.png)